src/momento/requests/collection_ttl.py

```diff
@@ -82,19 +84,20 @@ def refresh_ttl_if_provided(ttl: Optional[timedelta] = None) -> CollectionTtl:
         return CollectionTtl(ttl=ttl, refresh_ttl=ttl is not None)
 
     def with_refresh_ttl_on_updates(self) -> CollectionTtl:
-        """Specifies that the TTL for the collection should be refreshed when
-        the collection is modified.  (This is the default behavior.)
+        """Specifies the TTL for the collection be refreshed when the collection is modified.
+
+        NB: This is the default behavior.
```

Comment:
I had to look up what NB meant. ESL users will probably have trouble, too.

```suggestion
        This is the default behavior for a CollectionTTL.
```